docs/src/Provider.doc.js

```diff
@@ -19,65 +19,179 @@ card(
+        type: `'light' | 'dark' | 'userPreference'`,
         defaultValue: 'light',
-        description:
-          'The color scheme for components inside the provider. Specify "userPreference" to use "prefers-color-scheme" media query.',
-        href: 'colorScheme',
+        description: `The color scheme for components inside the provider. Specify 'userPreference' to use 'prefers-color-scheme' media query.`,
+        href: 'Color-scheme',
       },
       {
         name: 'id',
         type: 'string',
         description:
-          'An optional id for your provider. If not passed in, settings will be applied as globally as possible (example: it sets color scheme variables at :root).',
+          'An optional id for your color scheme provider. If not passed in, settings will be applied as globally as possible (example: it sets color scheme variables at :root).',
+        href: 'Color-scheme',
+      },
+      {
+        name: 'onNavigation',
+        type:
+          '{ href: string, onNavigationOptions:  ({ [string]: Node | ({| +event: SyntheticEvent<> |}) => void }) }',
+        description: [
+          'Components with link functionality use simple `<a>` tags. In order to replace the default link functionality with more complex ones (ex. withRouter or spam checking), onNavigation provides an interface to implement external logic into the onClick event handler in links.',
+          ` 'onNavigation' is a high-order function. If passed into the provider, consumer components (ex. Link, Button, IconButton, and TapArea) call the 'onNavigation' function with 2 named parameters ('href' and 'onNavigationOptions') that returns a function that gets called inside the 'onClick' event handler. 'onNavigationOptions' is an object that acts as a flexible API for your onNavigation external logic.`,
+        ],
+        href: 'OnNavigation',
       },
     ]}
   />,
 );
 
 card(
-  <Example
-    description="Specify a light or dark color scheme for components"
-    name="Color scheme"
-    id="colorScheme"
-    defaultCode={`
+  <MainSection name="Variants">
+    <MainSection.Subsection
+      title="Color scheme"
+      description="Specify a light or dark color scheme for components"
+    >
+      <MainSection.Card
+        cardSize="lg"
+        defaultCode={`
 function Example(props) {
   const [scheme, setScheme] = React.useState('light')
   const schemeOptions = [
     {
-      value: "light",
-      label: "Light"
+      value: 'light',
+      label: 'Light'
     },
     {
-      value: "dark",
-      label: "Dark"
+      value: 'dark',
+      label: 'Dark'
     },
     {
-      value: "userPreference",
-      label: "User Preference"
+      value: 'userPreference',
+      label: 'User Preference'
     }
   ];
   return (
-    <Provider colorScheme={scheme} id="docsExample">
-      <Box color="white" padding={2}>
+    <Provider colorScheme={scheme} id='docsExample'>
+      <Box color='white' padding={2}>
```

Comment:
These should remain double quotes

Reply:
DONE

Reply:
I'm still seeing single quotes?

Reply:
You better get drunk and see double! Come down to Florida! it's always 5 in here.